{"name": "test_auth_without_one_credential_returns_error_status_code[False-True]", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "exclude_login = False, exclude_password = True\n\n    @pytest.mark.parametrize(\n        \"exclude_login,exclude_password\",\n        [\n            (True, False),\n            (False, True)\n        ]\n    )\n    def test_auth_without_one_credential_returns_error_status_code(exclude_login, exclude_password):\n        credentials = get_registered_courier_credentials(exclude_login=exclude_login, exclude_password=exclude_password)\n        response = login_courier(credentials)\n>       verify_response_code(response, 400)\n\ntests/test_login_courier.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [504]>, code = 400\n\n    @allure.step('Проверяем код ответа сервера')\n    def verify_response_code(response: requests.Response, code: int):\n>       assert response.status_code == code\nE       AssertionError\n\nmethods.py:88: AssertionError"}, "steps": [{"name": "Получаем пару логин-пароль существующего курьера", "status": "passed", "parameters": [{"name": "exclude_login", "value": "False"}, {"name": "exclude_password", "value": "True"}, {"name": "corrupt_password", "value": "False"}], "start": 1704908987242, "stop": 1704908987242}, {"name": "Залогин курьера [POST]", "status": "passed", "parameters": [{"name": "credentials", "value": "{'login': 'login-test-777'}"}], "start": 1704908987242, "stop": 1704909047419}, {"name": "Проверяем код ответа сервера", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/Users/lkitova/arcadia/games/qa/qa_tests/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 173, in impl\n    return func(*a, **kw)\n  File \"/Users/lkitova/Documents/Sprint_7/methods.py\", line 88, in verify_response_code\n    assert response.status_code == code\n"}, "parameters": [{"name": "response", "value": "<Response [504]>"}, {"name": "code", "value": "400"}], "start": 1704909047420, "stop": 1704909047420}], "parameters": [{"name": "exclude_login", "value": "False"}, {"name": "exclude_password", "value": "True"}], "start": 1704908987242, "stop": 1704909047420, "uuid": "62c6e5f6-571a-4f15-b379-ed677645afc8", "historyId": "2a6f3bbc97f790a3673cbb9f26148c30", "testCaseId": "470bb0f55ed265d2ed8568132ba8244c", "fullName": "tests.test_login_courier#test_auth_without_one_credential_returns_error_status_code", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "host", "value": "i111355119"}, {"name": "thread", "value": "32285-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}